### PR TITLE
[tests] Add test case for DHCP APIs with websocket

### PR DESCRIPTION
**Pull request by cuiyanx**
**TOTAL:** 1 of 1 passed

Signed-off-by: Cui Yan <yanx.cui@intel.com>

### CODE REVIEW

File: tests/test-dhcp-ws-manual.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+if (board.name === "arduino_101") {
+    var bleAddr = "F1:E2:D3:C4:B5:A6";
+    netConfig.setBleAddress(bleAddr);
+
+    console.log("\nPlease connect a101 with Bluetooth....");
+    console.log("setBleAddress(bleAddr): expected result 'F1:E2:D3:C4:B5:A6'\n");
+
+    netConfig.on("netup", function() {
+        console.log("'onup' Event: expected result 'Connected'\n");
```
Comment (jimmy-huang) [inline tests/test-dhcp-ws-manual.js]: onnetup?

File: tests/test-dhcp-ws-manual.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+            clientTracking: true,
+            maxPayload : 125,
+            acceptHandler: ServerHandler
+        });
+
+        SocketBody(WSServer);
+    });
+
+    netConfig.on("netdown", function() {
+        console.log("'ondown' Event: expected result 'Disconnected'\n");
```
Comment (jimmy-huang) [inline tests/test-dhcp-ws-manual.js]: onnetdown

File: tests/test-dhcp-ws-manual.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+                    console.log("\nPlease close ws client");
+                }
+            } else {
+                websocket.send(new Buffer("DHCP"), true);
+                console.log("Send data: DHCP");
+            }
+        });
+
+        websocket.on("close", function(code, reason) {
+            console.log("ReceiveClose: " + code + " : " + reason);
```
Comment (jimmy-huang) [inline tests/test-dhcp-ws-manual.js]: Received close or 'closed'?

**Comment by grgustaf** [comment]
+1

**Pull request merged**